config/crd/bases/skupper_router_access_crd.yaml

```
@@ -52,11 +100,24 @@ spec:
               type: object
               properties:
                 status:
+                  description: |-
+                    The current state of the resource.
+                    `Pending`: The resource is being processed.
```

```suggestion
                   - `Pending`: The resource is being processed.
```
bullet all items in list